### ISSUE TITLE
Correct bug in handling of GATEWAY_SYSTEM_NAME with spaces

### ISSUE THREAD

**Issue by thirdgen88** [thirdgen88/ignition-docker#67]
**Description**
Error occurs when attempting to provide GATEWAY_SYSTEM_NAME with spaces:

**Expected Behavior**
- Gateway System Name provided via environment that contains spaces is processed correctly

**Actual Behavior**
- Error produced during entrypoint processing and gateway system name is not applied:
```
/usr/local/bin/docker-entrypoint.sh: line 35: [: node1: binary operator expected
```

**Comment by thirdgen88** [comment]
This should probably be fixed by #62..  Should also backport this to the 7.9 entry point.

**Comment by thirdgen88** [comment]
This was fixed with PR #71.  The rebuilt `8.1.7` tag has the updates as well as `7.9.18`.
